js/script.js

```diff
@@ -0,0 +1,269 @@
+let outfits = [{
+  outfitId: 1,
+  imgSrc: 'images/1.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 2,
+  imgSrc: 'images/2.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 3,
+  imgSrc: 'images/3.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 4,
+  imgSrc: 'images/4.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 5,
+  imgSrc: 'images/5.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 6,
+  imgSrc: 'images/6.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 7,
+  imgSrc: 'images/7.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 8,
+  imgSrc: 'images/8.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 9,
+  imgSrc: 'images/9.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 10,
+  imgSrc: 'images/10.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 11,
+  imgSrc: 'images/11.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 12,
+  imgSrc: 'images/12.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+}
+];
+
+let l = 0;
```

Comment:
I think you can come up with better names then `l` and 'r' because they are not really descriptive. Even `leftPic` and `rightPic` would be better.

Reply:
Fixed.